### DISCUSSION TITLE
Nothing happens when exporting to .h5

### DISCUSSION THREAD

**Discussion by juliagorman** [talmolab/sleap#1299] [Help!]
When I try to export my analysis to an h5 file on one video nothing happens. I checked and I am able to do it with other videos, but specifically not this one.

**Comment by roomrys** [comment]
Hi @juliagorman,

Two follow-up questions:

Are there any messages in your terminal? That might help guide us to the problem.

Also, is the analysis file being written and just empty (or an unexpected shape)?

Thanks,
Liezl

**Comment by juliagorman** [comment]
no messages are popping up in the terminal and no file at all is being written
